Tests/LibWeb/CMakeLists.txt

```diff
@@ -1,2 +1,13 @@
+set(
+    TEST_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/TestHTMLTokenizer.cpp
+)
+
+foreach(source ${TEST_SOURCES})
+    serenity_test(${source} LibWeb LIBS LibWeb)
+endforeach()
+
+# test-web does not use LibTest.
```

Comment:
I mean...it does, it just uses a different interface (see the definition of `serenity_testjs_test()`)